controllers/configs.go

```diff
@@ -11,8 +11,8 @@ import (
 	"reflect"
 	"time"
 
+	controlplanev1 "github.com/siderolabs/cluster-api-control-plane-provider-talos/api/v1alpha3"
 	cabptv1 "github.com/talos-systems/cluster-api-bootstrap-provider-talos/api/v1alpha3"
-	controlplanev1 "github.com/talos-systems/cluster-api-control-plane-provider-talos/api/v1alpha3"
```

Comment:
I think we shouldn't update this yet... probably we would rather should do a big migration for all repos at once, as it will be hard to remember which one migrated